src/SourceLocator/LocatedSource.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace BetterReflection\SourceLocator;
+
+/**
+ * Value object containing source code that has been located
+ */
+class LocatedSource
+{
+    /**
+     * @var string
+     */
+    private $source;
```

Comment:
`source` is source code itself, `filename` is a potential filename (may be null actually, so will update the doc)